gordo_components/builder/local_build.py

```diff
@@ -25,6 +28,11 @@ def local_build(config_str: str) -> Iterable[Tuple[Union[BaseEstimator, None], d
     ----------
     config_str: str
         The raw yaml config file in string format.
+    enable_mlflow: bool
+        Flag to enable Mlflow logging of model building results.
```

Comment:
Should this not be inferred from any `workspace_kwargs`?

Reply:
I did it like this, as you could potentially do a local build and log to a different remote workspace.

ie.
`enable_mlflow=True, workspace_kwargs={}` -> local logging
`enable_mlflow=True, workspace_kwargs={workspace_id, ...}` -> remote logging
`enable_mlflow=False, workspace_kwargs={} or {workspace_id, ...}` -> no logging

Reply:
I added some extra explanation to the docstring to clarify that a bit :+1:

Reply:
Ok, so it seems tied to a concept of `workspace` for remote logging. Otherwise I would have imagined that it would be something like `logging_kwargs` or something which could determine the difference. Anway, just ignore my comment. I think _that_ would involve much more changes.